index.html

```diff
@@ -291,6 +291,41 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <section class="form-section">
+      <div class="cols">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+
+      <form class="form-content" action="https://formspree.io/f/xknylojk" method="post">
+        <input id="full-name" name="full-name" maxlength="30" placeholder="Full Name" type="text" required />
```

Comment:
- [ ] Great job making the mobile version of this section identical to that of the **_Figma_** template you chose. However, I noticed that the layout/styling for the **_desktop_** version is different. On Figma, the **_desktop_** version has two fields for First and Last names whereas yours has only one field for the full names of users. Kindly make your design of the contact form for the **_desktop_** version identical to that of the Figma design you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/203836886-5a7620a2-b83d-4508-8c7c-a32ee51db884.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/203836841-56706cab-8cbc-44f5-a67c-63a766c33666.png)